proto/grpc_gateway.bzl

```diff
@@ -0,0 +1,40 @@
+GOOGLEAPIS_GOOGLE_API_BUILD_CONTENTS = """
```

Comment:
Echoing the comment above, this should be possible with `http_archive` with an `overlay`. I don't think this file will be needed.

Reply:
but still defined within rules_go in the repository.bzl right?